contracts/leverage_trading/src/cancel_order.rs

```diff
@@ -75,6 +74,14 @@ trait ContractCallbackInterface {
         repay_amount: U128,
     ) -> PromiseOrValue<U128>;
     fn withdraw_callback(&mut self, account_id: AccountId, token: AccountId, amount: U128);
+    fn cancel_take_profit_order_callbal(
```

Comment:
callbal=>callback

Reply:
`cancel_take_profit_order_callbal` and `cancel_take_profit_order`  in `trait ContractCallbackInterface` it's unnecessary code.
Removed